modules/backend/formwidgets/colorpicker/assets/js/colorpicker.js

```diff
@@ -57,24 +57,31 @@
                 chooseText: $.oc.lang.get('colorpicker.choose', 'Ok'),
                 cancelText: '⨯',
                 hide: function(color) {
-                    var hex = color ? color.toHexString() : ''
-                    self.$customColorSpan.css('background', hex)
+                    var customColor = self.covertColor(color)
+                    self.$customColorSpan.css('background', customColor)
                 },
                 show: function(color) {
                     self.selectColor(self.$customColor)
                 },
                 move: function(color) {
-                    var hex = color ? color.toHexString() : ''
-                    self.$customColorSpan.css('background', hex)
+                    var customColor = self.covertColor(color)
+                    self.$customColorSpan.css('background', customColor)
                 },
                 change: function(color) {
-                    var hex = color ? color.toHexString() : ''
-                    self.setCustomColor(hex)
+                    var customColor = self.covertColor(color)
+                    self.setCustomColor(customColor)
                 }
             })
         }
     }
 
+    /**
+     * Convert colour to hex or rgba depending on alpha (opacity) value
+     */
+    ColorPicker.prototype.covertColor = function(color) {
```

Comment:
typo, function name should be `convert`, not `covert`, unless you're saying that the color is now a spy.

Reply:
I fixed this, no conspiracy anymore